### DISCUSSION TITLE
inacccessible or not found

### DISCUSSION THREAD

**Discussion by jl69** [termux/termux-packages#7331] [Q&A]
hello,
i downloaded termux apk on my phone and follow a tuto in order to solve an issue.
i need to run the following command : props.

the result is : "inacccessible or not found".
can you please help me to run this ?
thank you ,

**Comment by ghost** [comment]
Are you sure that you are trying to run the correct command?

`props` has never existed in Termux and as far I know, there no package exist that provides utility `props` at all (at least Ubuntu doesn't have package providing command `props`). On Android you have `getprop` and `setprop`, maybe one of these is actually what you are looking for.
